content/en/docs/setup/independent/high-availability.md

```diff
@@ -160,141 +109,126 @@ SSH is required if you want to control all nodes from a single machine.
         apiVersion: kubeadm.k8s.io/v1beta1
         kind: ClusterConfiguration
         kubernetesVersion: stable
-        apiServer:
-          certSANs:
-          - "LOAD_BALANCER_DNS"
```

Comment:
Is this SAN generated automatically now? I know in the past it wasn't.

Reply:
yes, the code is [here](https://github.com/kubernetes/kubernetes/blob/df2094b3d728bd58c5f23b41add109e32fc7c301/cmd/kubeadm/app/util/pkiutil/pki_helpers.go#L386)